staff_features/resources/step_definitions/resource_shared.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+When 'the user filters by text with the Resource title' do
+  fill_in 'Filter by text', with: @uuid
+
+  find('#filter-text').send_keys(:enter)
+
+  rows = []
+  checks = 0
+
+  while checks < 5
+    checks += 1
+
+    begin
+      rows = all('tr', text: @uuid)
+    rescue Selenium::WebDriver::Error::JavascriptError
+      sleep 1
+    end
+
+    break if rows.length == 1
+  end
+end
+
+Then 'the Resource is in the search results' do
+  expect(page).to have_css('tr', text: @uuid)
+end
+
+Then 'the Resource view page is displayed' do
+  expect(find('h2').text).to eq "Resource #{@uuid} Resource"
```

Comment:
I think adding another constraint here is better than just a heading (I know the existing tests often look for only a heading), maybe expecting `'.record-toolbar button', text: Edit`, or expecting the exact URL, etc.

Reply:
Walking this suggestion back based on https://github.com/archivesspace/e2e-tests/pull/41#discussion_r1892359072